easybuild/easyblocks/m/mcr.py

```diff
@@ -137,6 +133,8 @@ def make_module_extra(self):
         """Extend PATH and set proper _JAVA_OPTIONS (e.g., -Xmx)."""
         txt = super(EB_MCR, self).make_module_extra()
 
+        if not self.subdir: self.find_subdir()
```

Comment:
please change this to:

```python
if not self.subdir:
    self.find_subdir()
```

(it's correct, but doesn't match the usual code style)

alternatively: just call `self.set_subdir()` and bump out early if `self.subdir` is already defined